source/guides/getting-started/quick-start.html.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Guides | Quick Start
+next_guide:
+  title: Interactors Introduction
+  url: /guides/interactors/introduction.html
+---
+
+# Quick Start
+
+This guide will teach you how to add BigTest to your React application.
+
+We'll cover:
+- Installing BigTest
+- Generating support files
+- Set Up
+- Writing some basic tests
```

Comment:
I ❤️TOCs. Even if it's a short document, it helps to have links directly to the different sections.

The website automatically adds `id`s to headlines so that they're linkable. So for the first bullet point, you could link directly to that headline like so: `[Installing BigTest](#install-dependencies)`.